examples/Authentication/AuthenticateInWebApplication.php

```diff
@@ -59,6 +59,11 @@ class AuthenticateInWebApplication
 
     public static function main()
     {
+        if (!class_exists(Server::class)) {
```

Comment:
Can you refer to the class using FQCN?

Reply:
IMO you import this class anyway (to use it later), so no need for FQCN. It also works without any problems when the library is not loaded and the class is not available.

But if you really want the change, I will do it, of course.

Reply:
That's fine. Yes, I already did the import above in this class. :)